### ISSUE TITLE
Lsar

### ISSUE THREAD

**Issue by LunarSera** [Kwoth/NadekoBot#2035]
Okay so this is an ongoing issue.
Started way back when who knows. Probably like a year ago.
Embeds are broke. How?
Well when I used to send lsar on a mobile device, Android specifically, well I would get 20 roles and that's it.
On desktop it would show correctly. If I had 30 roles it would show 30 roles.
Now I've confirmed on 2.13.8 that it only shows 20 roles on desktop too. Kinda useless don't you agree?

https://cdn.discordapp.com/attachments/312761960573698050/406312117369700352/unknown.png

This issue is quite annoying and I have no idea why it's not fixed. I mentioned it months ago and kwoth said it was on discords end.

Well please either look into it or paginate lsar for 15 roles.

Edit let me rephrase that I missed the change log for pagination. And that it does work on mobile. But from the code I would assume there are supposed to be reaction emojiis to flip pages

**Comment by LunarSera** [comment]
Looks like the pagination works but there are no reaction emojiis. Did I miss that?

**Comment by Kwoth** [comment]
No, there are no emojis. It works as intended.
